src/directives/until.ts

```diff
@@ -27,6 +27,11 @@ interface AsyncState {
 
 const _state = new WeakMap<Part, AsyncState>();
 
+function isThenable(value: unknown): value is PromiseLike<unknown> {
```

Comment:
I pretty uniformly prefer const arrow functions

Reply:
Interesting. const arrow functions produce slightly smaller bundled output.

I'll meet you halfway: const arrow functions with braces. Without the braces it's slightly harder to skim, and it seems to have a 0 byte effect on the output.